include/seqan/arg_parse/arg_parse_argument.h

```diff
@@ -1086,6 +1072,16 @@ inline std::string getFileExtension(ArgParseArgument const & me, unsigned pos =
     return value.substr(dotPos + 1);
 }
 
+// ----------------------------------------------------------------------------
+// Function isBooleanOption()
+// ----------------------------------------------------------------------------
+
+// needed for easy printing of the help page
+constexpr bool isBooleanOption(ArgParseArgument const & /*me*/)
```

Comment:
Its used here: https://github.com/seqan/seqan/blob/master/include/seqan/arg_parse/arg_parse_doc.h#L723
So instead of doubling the function _addDefaultValues() I thought I introduced isBooleanOption() for ArgParseArgument.